fvm/env.go

```diff
@@ -86,6 +86,8 @@ type commonEnv struct {
 	logs          []string
 	rng           *rand.Rand
 	traceSpan     opentracing.Span
+
+	hasFrozenAccounts bool
```

Comment:
nitpicking, but I find the name misleading. Its only true if the freezing has occurred, not if there are frozen accounts. `hasAccountFrozen` ? `didFreeze` ?